.github/workflows/publish-commit-bottles.yml

```diff
@@ -73,6 +75,38 @@ jobs:
         with:
           signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
 
+      - name: Check PR branch for mergeability
+        id: pr-branch-check
+        working-directory: ${{steps.set-up-homebrew.outputs.repository-path}}
+        env:
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        run: |
+          pr_data="$(
+            gh api \
```

Comment:
This is fine for now but feels like something it'd be nice to pull into Homebrew/brew at some point so it can be more easily tested.

Reply:
Yea, good point. I think we can pull this into `pr-publish`.